network-api/networkapi/wagtailpages/pagemodels/publications/article.py

```diff
@@ -88,9 +89,15 @@ def breadcrumb_list(self):
         """
         return Page.objects.ancestor_of(self).type(PublicationPage).live()
 
+    @property
+    def zen_nav(self):
```

Comment:
Adds an overwritable property to "trick" the templates into thinking this page has "zen_nav" as a property.

Reply:
see my other comment